package.json

```diff
@@ -26,12 +26,14 @@
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
-    "wp:dev": "cross-env NODE_ENV=development webpack",
-    "wp:prod": "cross-env NODE_ENV=production webpack",
-    "watch": "gulp watch",
+    "wp:dev": "cross-env NODE_ENV=development webpack --progress --colors",
+    "wp:prod": "cross-env NODE_ENV=production webpack --progress --colors",
+    "watch:gulp": "gulp watch",
+    "watch:wp": "cross-env NODE_ENV=development webpack --progress --colors --watch",
+    "watch": "npm-run-all --parallel watch:*",
     "test": "mocha",
     "test-gui": "node scripts/test-gui.js",
-    "lint": "eslint \"./source/**/*.js\" -f html -o eslint_report.htm"
+    "lint": "eslint --ext js,vue -f html -o eslint_report.htm source resources"
```

Comment:
Can we put the additional parameters into the `.eslintrc`?

Reply:
Target files have to be specified via command line arguments (the `eslintrc` is only to define the style): https://github.com/eslint/eslint/issues/11223

Reply:
Alright. But: Why specify both source and resource directory? I'm pretty sure I did this in the eslintrc, … right? I have no overview right now

Reply:
The eslintrc doesn't contain anything about the resource folder, which I added because it contains the vue files, some other js.

Reply:
Ah okay … but seriously, now I'm wondering why it did lint the source-directory correctly o_O

Reply:
The compiled files are ignored from linting using the `eslintignore` file.

Reply:
Ahh, alright. Thanks!